tests/src/contexts/UnitContext.test.tsx

```
@@ -31,9 +32,28 @@ describe("UnitContext", () => {
+                id: "test",
+                username: "test",
+                email: "test@example.com",
+                email_verified: true,
+                created_at: "2023-01-01T00:00:00Z",
+                updated_at: "2023-01-01T00:00:00Z",
+                is_active: true,
+              }
+            : null,
+          isAuthenticated,
+        },
+        children: React.createElement(UnitProvider, {
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use additional arguments to React.createElement

(lint/correctness/noChildrenProp)